ops/go.mod

```diff
@@ -243,3 +243,6 @@ require (
 
 // Usage of this gogo/protobuf fork is required by Cosmos SDK: https://github.com/cosmos/cosmos-sdk/issues/10702
 replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
+
+// Notice: uncomment while developing
+// replace github.com/smartcontractkit/chainlink-relay/ops => ../../chainlink-relay/ops
```

Comment:
unsure about how helpful this is to leave in

Reply:
Yeah, I can remove it. Could be helpful to have a quick local setting as we do interact x-repo a lot (chainlink, chainlink-relay, integrations-framework), but maybe a better place would be README.